release_gap.sh

```diff
@@ -1,6 +1,8 @@
+#!/bin/sh
+
 make --trace \
-    SAGE_GIT=https://www.github.com/ChrisJefferson/gap \
-    SAGE_VERSION=cygwin-fixes \
+    SAGE_GIT?=https://www.github.com/gap-system/gap \
```

Comment:
```suggestion
    SAGE_GIT?=https://github.com/gap-system/gap \
```
I wasn't even aware that using the `www` subdomain works. But the canonical name is without the `www.` (it immediately redirects if you add it)